docs/kyma/docs/031-gs-local-installation.md

```diff
@@ -40,6 +40,9 @@ Follow these steps to "always trust" the Kyma certificate on Mac:
 
 >**NOTE:** "Always trusting" the certificate does not work with Mozilla Firefox.
 
+>**NOTE:** To access the Application Connector and connect an external solution to the local deployment of Kyma, you must add the certificate to the trusted certificate storage of your programming environment. Read the **Access Application Connector on local Kyma** in the **Application Connector** topic to learn more.
```

Comment:
Another note? Maybe just add this paragraph as it is or connect it somehow (introductory sentence?) with the whole certificates section?